packages/core/src/mpi/get-patient-by-demo.ts

```diff
@@ -49,9 +53,13 @@ export const getPatientByDemo = async ({
     [matchingPersonalIdentifiersRule, matchingContactDetailsRule],
     foundPatients.map(patientToPatientMPI),
     normalizedPatientDemo,
-    SIMILARITY_THRESHOLD
+    SIMILARITY_THRESHOLD,
+    false
```

Comment:
to confirm, why no longer greedy? 
Because we have useFirstMatchingPatient so its redundant?

Reply:
Yes, `useFirstMatchingPatient` achieves the same result as greedy (albeit a bit less performant), and by switching to `false` we can also then flag the case when more than more patient is returned from `matchPatients`, which means `getPatientByDemo` is now making an a choice about who to return.

Reply:
yup ok makes sense